firmware/rhizome.ino

```diff
@@ -7,6 +7,7 @@
 #include "Ohmbrewer_Publisher.h"
 #include "Ohmbrewer_Thermostat.h"
 #include "Ohmbrewer_RIMS.h"
+#include "Ohmbrewer_Onewire.h"
 #include "onewire.h"
```

Comment:
You probably also want to add an ``` #include <list> ``` on line 19. The only reason it isn't yelling about this must be that we include list within the equipment.